airflow/api_connexion/endpoints/dataset_endpoint.py

```diff
@@ -311,3 +323,37 @@ def delete_dataset_queued_events(
         "Queue event not found",
         detail=f"Queue event with dataset uri: `{uri}` was not found",
     )
+
+
+@security.requires_access_dataset("POST")
+@provide_session
+@action_logging(
+    event=action_event_from_permission(
+        prefix=RESOURCE_EVENT_PREFIX,
+        permission=permissions.ACTION_CAN_CREATE,
+    ),
+)
+def post_dataset_event(session: Session = NEW_SESSION) -> APIResponse:
+    """Post dataset event."""
```

Comment:
i think this function should have a better name.  POST is the http method that is connected to this function.  but it doesn't tell us what this method does.  

i assume what it does is `create_dataset_event`?  that would seem to be a better name

Reply:
Yep, sounds good. Updated it. Thanks!